README.md

```diff
@@ -641,6 +641,7 @@ Table of Contents
   * [jare.io](http://www.jare.io) — CDN for images. Uses AWS CloudFront
   * [jsdelivr.com](https://www.jsdelivr.com/) — CDN of OSS (JS, CSS, fonts) for developers and webmasters, accepts PRs to add more
   * [Microsoft Ajax](https://docs.microsoft.com/en-us/aspnet/ajax/cdn/overview) — The Microsoft Ajax CDN hosts popular third-party JavaScript libraries such as jQuery and enables you to easily add them to your Web application
+  * [yastatic.net](https://yandex.ru/dev/jslibs/) — CDN for jQuery, BEM, lodash and more
```

Comment:
Thanks, I get what this is saying but as a rule we only accept pages written in english.  If there's a english version of this pls update the url and will take a look